pysipp/agent.py

```diff
@@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
     configuration options for a SIP UA.
     """
     # we skip `error` since we can get it from stderr
-    _log_types = 'screen calldebug message log'.split()
+    _log_types = 'screen calldebug message log error'.split()
```

Comment:
I just fixe the tests by updating pytest to version 2.9.2.

Now all tests pass. I am using SIPp 3.5.1.

Le 25/07/2016 à 17:30, goodboy a écrit :
>
> In pysipp/agent.py 
> <https://github.com/SIPp/pysipp/pull/19#discussion_r72086009>:
>
> > @@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
> >      configuration options for a SIP UA.
> >      """
> >      # we skip `error` since we can get it from stderr
> > -    _log_types = 'screen calldebug message log'.split()
> > +    _log_types = 'screen calldebug message log error'.split()
>
> @fchareyron <https://github.com/fchareyron> You'll have to post the 
> full traceback as I'm not entirely sure what's going wrong. Also note 
> you can syntax highlight code blocks here in Github comments using 
> these instructions 
> <https://help.github.com/articles/creating-and-highlighting-code-blocks/>.
>
> Basically what we want to see (in order to get your changes into the 
> projects) is that the unit test suite passes. If you need to modify 
> the tests that is fine as long as it makes sense and is reliable.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/SIPp/pysipp/pull/19/files/271d6193be712c3342e3f137cd4c678aee6cf75c#r72086009>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AToJTQ9uyubWjcxzSk_YREfw86mYpaN0ks5qZNaTgaJpZM4JSigv>.
>